ee/fleetctl/updates.go

```diff
@@ -29,7 +29,7 @@ const (
 	// Expirations from
 	// https://github.com/theupdateframework/notary/blob/e87b31f46cdc5041403c64b7536df236d5e35860/docs/best_practices.md#expiration-prevention
 	// ~10 years
-	rootExpirationDuration = 10 * 365 * 24 * time.Hour
+	rootExpirationDuration = 10 * 365 * 24 * time.Hour //nolint:unused,deadcode
```

Comment:
Even if not used, it seemed like we may want to keep it as part of the list of expiration constants.